scripts/install_operator.sh

```diff
@@ -6,9 +6,15 @@ SPECS_DIR="$(dirname ${SCRIPT_DIR})/specs"
 
 NAMESPACE=${NAMESPACE:-spark-operator}
 
-OPERATOR_NAME=${OPERATOR_NAME:-spark-operator}
+if [ -f "${SCRIPT_DIR}"/../operator-build ]; then
+  echo "operator-build file is found. Parse OPERATOR_IMAGE_NAME and OPERATOR_VERSION from there."
+  OPERATOR_IMAGE_NAME=$(awk -F':' '{printf $1}' <"${SCRIPT_DIR}"/../operator-build)
+  OPERATOR_VERSION=$(awk -F':' '{printf $2}' <"${SCRIPT_DIR}"/../operator-build)
```

Comment:
in case a user executes the following command, provided variables will be ignored in case file `operator-build` is present: 
```
OPERATOR_IMAGE_NAME=user/spark-operator OPERATOR_VERSION=latest ./scripts/install_operator.sh
```
can we rely on the values provided in Makefile so the script doesn't make assumptions about any files created by external tools and directories layout of the project?

Reply:
it's good point about override user defined env vars, thanks for catch it. Fixed.

> can we rely on the values provided in Makefile so the script doesn't make assumptions about any files created by external tools and directories layout of the project?

make file generates tag versions in runtime, no chance to get it without run some `make` target.

Reply:
but that's the whole point, isn't it? So here are the scenarios:
- a user runs `make install` and values are propagated
- a user runs `install_operator.sh` and defaults are applied
- a user runs `OPERATOR_IMAGE_NAME=user/spark-operator OPERATOR_VERSION=latest install_operator.sh` and gets her values propagated
- (OR) `make install OPERATOR_IMAGE_NAME=user/spark-operator OPERATOR_VERSION=latest`

so with the current approach what doesn't look great is that:
- the script is aware of some file and its location (rename the file, and the script stops working)
- the script is aware of the structure of this file and knows how to parse it (same problem as above)

So I'd suggest removing this newly added logic and rely on `make` or a user for passing necessary parameters at the moment of execution. If a user wants to rerun the script, she can run `make install` one more time. 

What we can do to improve it - is to remove the default values in the script and add a check for those variables presence. If variables are present - we add Helm/KUDO image parameters, if not - rely on defaults specified in Chart/Operator. 
WDYT?